### ISSUE TITLE
Pkgconfig errors after using app

### ISSUE THREAD

**Issue by brysgo** [Neo-Oli/termux-ubuntu#19]
I have recurring pkgconfig errors whenever I install things. One is with openssh and I can't recall the other.

**Comment by Neo-Oli** [comment]
I think it has to do with groupadd and other user and groups related function. Maybe some of them can be solved by creating empty stubs for the commands.

**Comment by brysgo** [comment]
Yeah, stubs should do, do you have any idea how we would package it? Have you ever created a custom apt source?

**Comment by Neo-Oli** [comment]
Hmm... I'm not sure we need to do something that complicated. Maybe we can simply create a folder with the stubs in them and then copy them to the right folder during installation?

**Comment by brysgo** [comment]
That sounds good, no need to over complicate

**Comment by Neo-Oli** [comment]
I created a branch with some stubs https://github.com/Neo-Oli/termux-ubuntu/tree/stubs,  can you please test it? I still can't install things like dbus without errors, but I don't know why it isn't working.

**Comment by brysgo** [comment]
I haven't noticed the stubs breaking anything yet, but they haven't fixed anything that I can find yet. I will have to take a closer look.

**Comment by diehard88** [comment]
is this the problem i am facing?

$ ./start.sh
groups: cannot find name for group ID 3003
groups: cannot find name for group ID 9997
groups: cannot find name for group ID 50251
mesg: change /dev/pts/1 mode failed: Permission denied

root@localhost:~# apt update
Hit:1 http://ports.ubuntu.com/ubuntu-ports artful InRelease
Hit:2 http://ports.ubuntu.com/ubuntu-ports artful-updates InRelease
Hit:3 http://ports.ubuntu.com/ubuntu-ports artful-backports InRelease
Hit:4 http://ports.ubuntu.com/ubuntu-ports artful-security InRelease
Reading package lists... Done
Building dependency tree
Reading state information... Done
1 package can be upgraded. Run 'apt list --upgradable' to see it.
root@localhost:~#
root@localhost:~#

root@localhost:~# apt list --upgradable
Listing... Done
libc-bin/artful-updates,artful-security 2.26-0ubuntu2.1 arm64 [upgradable from: 2.26-0ubuntu2]
N: There is 1 additional version. Please use the '-a' switch to see it

root@localhost:~# apt upgrade
Reading package lists... Done
Building dependency tree
Reading state information... Done
Calculating upgrade... Done
The following packages will be upgraded:
libc-bin
1 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
1 not fully installed or removed.
Need to get 0 B/2724 kB of archives.
After this operation, 1024 B of additional disk space will be used.
Do you want to continue? [Y/n] y
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
LANGUAGE = (unset),
LC_ALL = (unset),
LANG = "en_US.UTF-8"
are supported and installed on your system.
perl: warning: Falling back to the standard locale ("C").
debconf: delaying package configuration, since apt-utils is not installed
dpkg: error processing package libc6:arm64 (--configure):
package is in a very bad inconsistent state; you should
reinstall it before attempting configuration
Errors were encountered while processing:
libc6:arm64
E: Sub-process /usr/bin/dpkg returned an error code (1)
root@localhost:~#

**Comment by RupankarGhosh** [comment]
Works great on my artful....at first it showing like you mentioned....but after setting the Perl and the locales...the problem gone

**Comment by diehard88** [comment]
thanks...

Got it workin on one android.

adb package was successfully installed.
both androids can see each other through USB (point-to-point), however,  adb does not list any device.
note: me no root.

`a.out  downloads  nikola  storage  termux-ubuntu
$ cd termux-ubuntu/
$ ./start.sh
groups: cannot find name for group ID 3003
groups: cannot find name for group ID 9997
groups: cannot find name for group ID 50251
mesg: change /dev/pts/0 mode failed: Permission denied
root@localhost:~#
root@localhost:~#
root@localhost:~# adb start-server
* daemon not running. starting it now on port 5037 *
* daemon started successfully *
root@localhost:~#
root@localhost:~#
root@localhost:~#
root@localhost:~# adb devices
List of devices attached

root@localhost:~# adb devices -l
List of devices attached

root@localhost:~#
root@localhost:~#
root@localhost:~# adbd
root@localhost:~# adbd --help
Segmentation fault
root@localhost:~#`
